end-to-end-tests/end_to_end_tests.py

```diff
@@ -0,0 +1,277 @@
+import argparse
+
+import requests
+import json
+import base64
+import time
+from datetime import datetime, timedelta
+from pprint import pprint
+
+from stellar_sdk import (
+    Asset,
+    Keypair,
+    Network,
+    TransactionEnvelope,
+    TransactionBuilder,
+    Server,
+)
+from stellar_sdk.sep.federation import fetch_stellar_toml
+
+STELLAR_TESTNET_NETWORK_PASSPHRASE = "Test SDF Network ; September 2015"
+HORIZON_URI = "https://horizon-testnet.stellar.org"
+FRIENDBOT_URI = "https://friendbot.stellar.org"
+
+
+class Endpoints:
+    def __init__(self, domain, use_http=True):
+        toml = fetch_stellar_toml(domain, use_http=use_http)
+        self.ANCHOR_PLATFORM_AUTH_ENDPOINT = toml.get("WEB_AUTH_ENDPOINT")
+        self.ANCHOR_PLATFORM_SEP12_CUSTOMER_ENDPOINT = toml.get("KYC_SERVER") + "/customer"
+        self.ANCHOR_PLATFORM_SEP38_QUOTE_ENDPOINT = toml.get("ANCHOR_QUOTE_SERVER") + "/quote"
+        self.ANCHOR_PLATFORM_SEP31_TRANSACTIONS_ENDPOINT = toml.get("DIRECT_PAYMENT_SERVER") + "/transactions"
+
+
+def get_anchor_platform_token(endpoints, public_key, secret_key):
+    print("============= Getting Token from Anchor Platform ========================")
+    response = requests.get(endpoints.ANCHOR_PLATFORM_AUTH_ENDPOINT, {"account": public_key})
+    content = response.json()
+
+    envelope_xdr = content["transaction"]
+    envelope = TransactionEnvelope.from_xdr(
+        envelope_xdr, network_passphrase=STELLAR_TESTNET_NETWORK_PASSPHRASE
+    )
+    envelope.sign(secret_key)
```

Comment:
FYI: clients should check if the sequence number of this tx is `"0"` before signing it, so they are sure the transaction is not going to be accepted in the Ledger.

Not a blocker for your PR though because this is not what we're interested in testing here.

Reply:
thanks for pointing this out! This is good to know, i'll keep this in mind in the future